test/tracing.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  BasicTracerProvider,
+  InMemorySpanExporter,
+  SimpleSpanProcessor,
+} from '@opentelemetry/tracing';
+
+/**
+ * This file is used to initialise a global tracing provider and span exporter
+ * for our tests used in our project. This was the only way I was able to get
+ * the tracing tests to work.
+ *
+ * Now before each test related or touches Opentelemetry
+ * we are resetting the exporter defined below to ensure there are now spans
+ * from previous tests still in memory.
```

Comment:
Comment nitpick: did you mean 'there are no spans'? I stared at that for a while, because I think the intent is that any previous tracers/spans/etc are removed from previous tests. :)

Reply:
@feywind  I have updated the comment :)